From 7424f2bea0cb412e96202f596ad8077131589f40 Mon Sep 17 00:00:00 2001
From: Adam Duskett <Aduskett@gmail.com>
Date: Thu, 14 Jul 2016 13:18:24 -0400
Subject: [PATCH] Remove hardcoded arch variable.

Allow the ARCH value to be passed in as original configuration was
solely based on host architecture.

This patch was updated to work with version 2.5

Signed-off-by: Clayton Shotwell <clayton.shotwell@rockwellcollins.com>
Signed-off-by: Niranjan Reddy <niranjan.reddy@rockwellcollins.com>
Signed-off-by: Adam Duskett <AdamDuskett@outlook.com>
Signed-off-by: Adam Duskett <Aduskett@gmail.com>
---
 policycoreutils/mcstrans/src/Makefile   | 1 -
 policycoreutils/mcstrans/utils/Makefile | 1 -
 2 files changed, 2 deletions(-)

diff --git a/mcstrans/src/Makefile b/mcstrans/src/Makefile
index 6fda57e..7b4489f 100644
--- a/mcstrans/src/Makefile
+++ b/mcstrans/src/Makefile
@@ -1,4 +1,3 @@
-ARCH = $(shell uname -i)
 # Installation directories.
 PREFIX  ?= $(DESTDIR)/usr
 SBINDIR ?= $(DESTDIR)/sbin
diff --git a/mcstrans/utils/Makefile b/mcstrans/utils/Makefile
index 1ffb027..912fe12 100644
--- a/mcstrans/utils/Makefile
+++ b/mcstrans/utils/Makefile
@@ -2,7 +2,6 @@
 PREFIX ?= $(DESTDIR)/usr
 BINDIR ?= $(PREFIX)/sbin
 
-ARCH = $(shell uname -i)
 ifeq "$(ARCH)" "x86_64"
         # In case of 64 bit system, use these lines
         LIBDIR=/usr/lib64
-- 
2.7.4

